api/controller/users.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable indent */
```

Comment:
Remove this comment.
Also, changes in this file are irrelevant to the feature. Remove unnecessary changes.

Reply:
done